src/lib/db.ts

```diff
@@ -18,7 +18,7 @@ import {
 } from "../models/document/document-types";
 import { SectionModelType } from "../models/curriculum/section";
 import { SupportModelType } from "../models/curriculum/support";
-import { ImageModelType } from "../models/image";
+import { ImageModelType } from "../plugins/image/models/image";
```

Comment:
To be fair, there are a 3 places where the core is also depending on `plugins/drawing-tool`.   
One of those should be easy to clean up. The other 2 will be harder. 

This case doesn't look too hard to fix. It could just use its own ImageType